ct1/vehinhtron.cpp

```diff
@@ -0,0 +1,79 @@
+#include<iostream>
+
+using namespace std;
+
+string bankinh, toadox, toadoy;
```

Comment:
Không sử dụng biến toàn cục trừ khi cần thiết. Trong trường hợp này hoàn toàn có thể sử dụng biến cục bộ thay thế.